### DISCUSSION TITLE
Issue with changing-the-colors.md

### DISCUSSION THREAD

**Discussion by stephffuller** [squidfunk/mkdocs-material#7143] [General]
Something's off here right before the name of the `mkdocs.yml` file.
![image](https://github.com/squidfunk/mkdocs-material/assets/52290324/e154ad86-7134-4687-a739-3a9e3a407d6e)

**Comment by kamilkrzyskow** [comment]
What do you mean? The website seems fine to me: https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#custom-colors
The `===` syntax is added via markdown extensions generated with MkDocs, they're not compatible with GitHub Markdown processor, so the file might appear broken.
